test/unit/world_location_test.rb

```diff
@@ -161,13 +161,19 @@ class WorldLocationTest < ActiveSupport::TestCase
     assert_equal [link_6, link_2, link_1, link_4, link_3], world_location.featured_links.only_the_initial_set
   end
 
-  test "has removeable translations" do
-    stub_any_publishing_api_call
+  test "translated world locations appear to only have :en translated locales" do
+    world_location = create(:world_location, translated_into: [:fr, :en])
+    assert_equal [:en], world_location.translated_locales
+  end
+
+  test "translated world locations appear to only have :en available locales" do
```

Comment:
This is fixed now despite what github seems to think. This test is for `available_locales` (now I've fixed it). The one above is `translated_locales`